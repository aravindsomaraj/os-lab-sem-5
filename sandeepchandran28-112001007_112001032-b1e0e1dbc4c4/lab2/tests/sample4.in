# This file has 1 scalar.
# This file has 4 matrices.

200

2 2
1,2,
3,4,


2 3
1,2,3,
4,5,6,


3 2
1,2,
3,4,
5,6


2 2
1,2,
3,4,