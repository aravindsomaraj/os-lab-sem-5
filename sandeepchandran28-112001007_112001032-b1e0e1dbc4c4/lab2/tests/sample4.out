2 2
69988,102864,
70682,103884,